# 2.py

# NameError exception: for data input in Python2.x, but incorrect in
# Python3.x
a = raw_input('Enter a number: ')
print(a)
